rss_pipeline/load.py

```diff
@@ -0,0 +1,67 @@
+'''Extracts data from the rss news articles and populates the media-sentiment relational database (rds)'''
+
+from os import environ
+
+from dotenv import load_dotenv
+import pandas as pd
+from psycopg2 import errors, extensions, extras
+import psycopg2
+
+
+def db_connection() -> psycopg2.extensions.connection | None:
+    '''Establish connection with the media-sentiment rds'''
+    try:
+        return psycopg2.connect(dbname=environ["DATABASE_NAME"],
+                                user=environ["DATABASE_USERNAME"],
+                                host=environ["DATABASE_ENDPOINT"],
+                                password=environ["DATABASE_PASSWORD"],
+                                cursor_factory=psycopg2.extras.RealDictCursor)
+
+    except:
+        raise psycopg2.DatabaseError("Error connecting to database.")
+
+
+def create_dataframe(file_name: str) -> pd.DataFrame:
+    '''Creates a Pandas DataFrame with the contents of the provided csv file'''
+    rss_df = pd.read_csv(file_name)
+    return rss_df
+
+
+def get_source_id(conn: psycopg2.extensions.connection, url) -> str:
+    try:
+        source = url.lstrip('https://www.').split('.co.uk')[0]
```

Comment:
I'd recommend adding a unit test for this line as it will test the implementation of this code rather than `psycopg2`. You could potentially move this into another function e.g. `def extract_source_from_url(url: str) -> str:`